modules/javafx.graphics/src/main/java/com/sun/javafx/util/Utils.java

```diff
@@ -683,10 +681,7 @@ private static VPos getVPosOpposite(HPos hpos, VPos vpos) {
 
     public static boolean hasFullScreenStage(final Screen screen) {
         @SuppressWarnings("removal")
-        final List<Window> allWindows = AccessController.doPrivileged(
-                (PrivilegedAction<List<Window>>) () -> Window.getWindows(),
-                null,
-                ACCESS_WINDOW_LIST_PERMISSION);
```

Comment:
@kevinrushforth once we finish removing AccessController, do we need to remove `FXPermissions` class?

Reply:
Yes, that's my plan (and is already in progress).